requirements/tests-requirements.txt

```diff
@@ -12,5 +12,5 @@ pytest-asyncio
 pandas
 scikit-learn
 imageio>=2.5.0
-docker
```

Comment:
why removing docker?

Reply:
fs is already a bentoml dependency, no need to add it here in tests dependency

Reply:
we don't use docker anymore